src/interface/guide/components/Preparation/ExtraDivs.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from '@emotion/styled';
+import { RoundedPanel } from 'interface/guide/components/GuideDivs';
+import { QualitativePerformance } from 'parser/ui/QualitativePerformance';
+import { BadColor, GoodColor, OkColor, PerfectColor } from 'interface/guide/index';
+import { PropsWithChildren } from 'react';
+
+export const StartAlignedRoundedPanel = styled(RoundedPanel)`
+  align-content: start;
+`;
+
+export const RoundedPanelWithColorBoxShadow = styled(StartAlignedRoundedPanel)`
+  box-shadow: inset 0.5em 0 0 ${(props) => props.color};
+`;
+
+const qualitativePerformanceToColor = (qualitativePerformance: QualitativePerformance) => {
+  switch (qualitativePerformance) {
+    case QualitativePerformance.Perfect:
+      return PerfectColor;
+    case QualitativePerformance.Good:
+      return GoodColor;
+    case QualitativePerformance.Ok:
+      return OkColor;
+    case QualitativePerformance.Fail:
+      return BadColor;
+  }
+};
```

Comment:
I thought this function already existed in the guide index... if not, you should move it there because this should be a core Guide function